psi4/driver/qcdb/vib.py

```diff
@@ -529,6 +538,24 @@ def vec_in_space(vec, space, tol=1.0e-4):
     xL = np.sqrt(reduced_mass_u) * wL
     vibinfo['x'] = QCAspect('normal mode', 'a0', xL, 'normalized un-mass-weighted')
 
+    # IR intensities, CCQC Proj. Eqns. 15-16
+    uconv_kmmol = (qcel.constants.get("Avogadro constant") * np.pi * 1.e-3 * qcel.constants.get("electron mass in u") *
+                   qcel.constants.get("fine-structure constant")**2 * qcel.constants.get("atomic unit of length") / 3)
```

Comment:
Jeez.  I'm glad you figured this out.   No way I would've ever figured out the random factor of three or the fine structure constant squared.